test-runner/src/reporters/base.ts

```diff
@@ -131,7 +129,8 @@ export class BaseReporter implements Reporter  {
   formatFailure(test: Test, index?: number): string {
     const tokens: string[] = [];
     const relativePath = path.relative(process.cwd(), test.file);
-    const header = `  ${index ? index + ')' : ''} ${terminalLink(relativePath, `file://${os.hostname()}${test.file}`)} › ${test.title}`;
+    const passedUnexpectedlySuffix = test.results[0].status === 'passed' ? ' -- passed unexpectedly' : '';
+    const header = `  ${index ? index + ')' : ''} ${terminalLink(relativePath, `file://${os.hostname()}${test.file}`)} › ${test.title}${passedUnexpectedlySuffix}`;
```

Comment:
Unrelated, but why do we print this full path? I'd personally prefer a relative path plus a line number.

Reply:
I HAVE NO IDEA :)

Reply:
The full path is not printed. Its a relative path + a link to the full path so you can click it in your terminal.

Reply:
![image](https://user-images.githubusercontent.com/4624233/91526847-663ba480-e8b9-11ea-8660-5a7b57bc32b5.png)

Reply:
Well, I have some news for you 😄 
<img width="881" alt="Screen Shot 2020-08-28 at 7 50 05 AM" src="https://user-images.githubusercontent.com/9881434/91581317-28636e00-e903-11ea-8284-4caa4209ca85.png">